src/frame/js/components/Action.jsx

```diff
@@ -139,8 +140,8 @@ export class ActionComponent extends Component {
                                        email={ user.email }
                                        amount={ amount }
                                        currency={ currency.toUpperCase() }
-                                       name={ stripeAccount.appName }
-                                       image={ stripeAccount.iconUrl }
```

Comment:
Yes, this https://github.com/smooch/smooch-js/pull/582/commits/89faf6ba83303f89c2135857409b48bb8e79bf6e